go/engine/box_audit_scheduler_background.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
```

Comment:
2019

why do you need 2 engines here?

Reply:
One is for box auditing a random team every day or so. The other is for reauditing teams that failed an audit